locales/es.po

```diff
@@ -11,31 +11,31 @@ msgstr ""
 
 #: src/renderer/components/activate_online_form/activate_online_form.messages.ts:3:30
 msgid "2-4 uppercase English characters"
-msgstr ""
+msgstr "2-4 caracteres en inglés en mayúsculas"
```

Comment:
```suggestion
msgstr "2-4 caracteres en mayúsculas"
```

Saying "in english" in spanish shouldn't really be literally translated. It would cause confusion for users, even if the message is trying to asert that special character won't be accepted.